src/components/Modals/SetAclPermsDocContainerModal.jsx

```diff
@@ -0,0 +1,172 @@
+// React Imports
+import React, { useContext, useState } from 'react';
+// Custom Hook Imports
+import { useSession, useStatusNotification } from '@hooks';
+// Material UI Imports
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import ClearIcon from '@mui/icons-material/Clear';
+import Dialog from '@mui/material/Dialog';
+import FormControl from '@mui/material/FormControl';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import SettingsIcon from '@mui/icons-material/Settings';
+import TextField from '@mui/material/TextField';
+// Utility Imports
+import { runNotification, setDocContainerAclPermission } from '@utils';
+// Context Imports
+import { SignedInUserContext } from '@contexts';
+// Component Imports
+import FormSection from '../Form/FormSection';
```

Comment:
Line could be changed to `import { FormSection } from './Form'`

Reply:
What's the difference? Not in a disrespectful way, I don't actually know what the difference is.

Reply:
One's a named import importing from `src/components/Form/index.js` the other is a default import importing from `src/components/Form/FormSection.jsx`.

Depending on whether or not there's going to be additional changes or modifications from components inside the "src/component/Form", it might be more useful to use the named import approach. Seems like you've already changed it to the named import. Which should be fine.